src/tasks/AppleAppBuilder/Templates/CMakeLists.txt.template

```diff
@@ -66,12 +66,7 @@ endif()
 
 target_link_libraries(
     %ProjectName%
-    "-framework Foundation"
-    "-framework Security"
-    "-framework UIKit"
-    %FrameworksToLink%
-    "-lz"
-    "-lc++"
-    "-liconv"
```

Comment:
I just remembered that we also call into AppleAppBuilder from dotnet/samples: https://github.com/dotnet/samples/blob/main/core/mono-samples/iOS/iOSSampleApp.csproj#L31

(we didn't do a good job of keeping this working so far but we probably need to take a look and see if it still works with net8)

Reply:
do we really need to move these system libraries into the .targets?

Reply:
I think it's better if we move what we are linking into msbuild. While these probably won't change, it's a much better dev loop experience being able to edit the file and then run the test.  It's a multi-step process keeping it in the cmake template.